eng/scripts/docs/Docs-Onboarding.ps1

```diff
@@ -67,153 +67,144 @@ function Get-python-DocsPackagesAlreadyOnboarded($docRepoLocation, $moniker) {
-    New-Item -ItemType Directory -Force -Path $installTargetFolder | Out-Null
-    $packageExpression = "$packageName$packageVersion"
-    try {
-        $pipInstallOutput = ""
-        if ($PackageSourceOverride) {
-            Write-Host "python -m pip install $packageExpression --no-cache-dir --target $installTargetFolder --extra-index-url=$PackageSourceOverride"
-            $pipInstallOutput = python -m pip `
-                install `
-                $packageExpression `
-                --no-cache-dir `
-                --target $installTargetFolder `
-                --extra-index-url=$PackageSourceOverride 2>&1
-        }
-        else {
-            Write-Host "python -m pip install $packageExpression --no-cache-dir --target $installTargetFolder"
-            $pipInstallOutput = python -m pip `
-                install `
-                $packageExpression `
-                --no-cache-dir `
-                --target $installTargetFolder 2>&1
-        }
-        if ($LASTEXITCODE -ne 0) {
-            LogWarning "python -m pip install failed for $packageExpression"
-            Write-Host $pipInstallOutput
-            return $false
-        }
-    }
-    catch {
-        LogWarning "python -m pip install failed for $packageExpression with exception"
-        LogWarning $_.Exception
-        LogWarning $_.Exception.StackTrace
-        return $false
-    }
-
-    return $true
+# The package info format required by py2docfx to verify a single library.
+# The format can be found at https://github.com/MicrosoftDocs/azure-docs-sdk-python/blob/main/ci-configs/packages-latest.json
+# which is a file that contains all of package_info objects. For verifying a single library
+# the same format is used but there's only a single package_info object in the packages list.
+# This is an example:
+# {
+#   "packages": [
+#       {
+#           "package_info": {
+#             "name": "azure-core",
+#             "install_type": "pypi",
+#             "prefer_source_distribution": "true",
+#             "version": "==1.29.6a20231207001",
+#             "extra_index_url": "https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-python/pypi/simple/"
+#           },
+#           "exclude_path": [
+#             "test*",
+#             "example*",
+#             "sample*",
+#             "doc*"
+#           ]
+#         }
+#       ]
+#   }
+# The above example contains the common settings for Python track 2 libraries.
+# There are a few things of note:
+# 1. install_type can be something other than pypi. source_code or dist_file are two examples of this
+#    but those are for track 1 or libraries released by other teams and not through our engineering system.
+# 2. extra_index_url only needs to exist on the object if PackageSourceOverride is set
+# 3. The reason this needs to be done using a json file instead of just a command line is because py2docfx
+#    doesn't handle the autodoc_default_options on the
```

Comment:
nit: comment trailing off.

Reply:
Finished the sentence in the 12th and final commit.